frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/index.tsx

```diff
@@ -84,6 +117,19 @@ export default function ApplyFlowChildSummary() {
   const isSubmitting = fetcher.state !== 'idle';
   const hasChildren = children.length > 0;
 
+  function handleSubmit(event: SyntheticEvent<HTMLFormElement, SubmitEvent>) {
+    event.preventDefault();
+
+    const formData = new FormData(event.currentTarget, event.nativeEvent.submitter);
+    const childId = event.nativeEvent.submitter?.dataset.childId;
+
+    if (childId) {
+      formData.set('child-id', childId);
+    }
+
+    fetcher.submit(formData, { method: 'POST' });
+  }
+
```

Comment:
Use an input of type `hidden` to store the `childId` instead. Try to use `fetcher.Form` per child.

```
<input type="hidden" name="childId" value={child.id} />
```